screens/TimerScreen.js

```diff
@@ -30,7 +33,7 @@ export default function TimerScreen() {
             btnRound={10}
             id={() => (
                 <Text style={{ color: 'gray', textAlign: 'center' }}>
-                    Your ID: {extractID(Constants.deviceId)}{' '}
+                    Your name: {random_name({ seed:extractID(Constants.deviceId), first: true })}{' '}
```

Comment:
Not optimal code will slow render, because name is generated on each render

Reply:
Look comment above and replace `random_name({ seed:extractID(Constants.deviceId), first: true })` with `name`